app/models/manageiq/providers/oracle_cloud/cloud_manager.rb

```diff
@@ -29,8 +29,84 @@ def self.description
     @description ||= "Oracle Cloud".freeze
   end
 
-  def description
-    ManageIQ::Providers::OracleCloud::Regions.find_by_name(provider_region)[:description]
+  def self.params_for_create
+    {
+      :fields => [
+        {
+          :component    => "select",
+          :id           => "provider_region",
+          :name         => "provider_region",
+          :label        => _("Region"),
+          :isRequired   => true,
+          :validate     => [{:type => "required"}],
+          :includeEmpty => true,
+          :options      => provider_region_options
+        },
+        {
+          :component => 'sub-form',
+          :name      => 'endpoints-subform',
+          :id        => 'endpoints-subform',
+          :title     => _("Endpoints"),
+          :fields    => [
+            {
+              :component              => 'validate-provider-credentials',
+              :id                     => 'authentications.default.valid',
+              :name                   => 'authentications.default.valid',
+              :skipSubmit             => true,
+              :isRequired             => true,
+              :validationDependencies => %w[type zone_id provider_region],
+              :fields                 => [
+                {
+                  :component  => "text-field",
+                  :id         => "authentications.default.userid",
+                  :name       => "authentications.default.userid",
+                  :label      => _("User ID"),
+                  :helperText => _("Should have privileged access, such as root or administrator."),
+                  :isRequired => true,
+                  :validate   => [{:type => "required"}]
+                },
+                {
+                  :component  => "text-field",
+                  :id         => "authentications.default.service_account",
+                  :name       => "authentications.default.service_account",
+                  :label      => _("Tenant ID"),
+                  :isRequired => true,
+                  :validate   => [{:type => "required"}]
+                },
+                {
+                  :component      => "password-field",
+                  :componentClass => 'textarea',
```

Comment:
This doesn't appear to be working anymore.  We have to preserve the public/private key newlines and the standard text-field is stripping them out before submitting the form.

I pulled this `'textarea'` from the google provider (which takes a JSON object as its auth_key) but that doesn't seem to be working either anymore

cc @h-kataria

Reply:
This seems like it is broken across the board, openstack also has a `textarea` for a private_key and when I upload a key that looks like:
```
-----BEGIN PRIVATE KEY-----
MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQCh0Y/ldIDWZjNf
gbb0ZGvHOjtbWj3eKG+nBETptJkCtqDjJ/slX2E17JTkUYNnhbiCNR/RoY2HXETM
LbRSQ8BMniTwJJ4EzyIa3zV+hIIi9GbJFZYZlHlwN7CnAISSFHzowIzccIIFcA9q
...
-----END PRIVATE KEY-----
```

This is what gets saved:
```
>> ManageIQ::Providers::Openstack::CloudManager.first.authentications.last.auth_key
=> "-----BEGIN PRIVATE KEY----- MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQCh0Y/ldIDWZjNf  gbb0ZGvHOjtbWj3eKG+ ... -----END PRIVATE KEY-----
```

(Note no newlines)

Reply:
@agrare Can you open a general bug issue in ui-classic?  That way we can focus on that problem separately.

Reply:
:+1: https://github.com/ManageIQ/manageiq-ui-classic/issues/7647